app/qml/ProjectIssuesPanel.qml

```diff
@@ -21,7 +21,7 @@ Item {
   property real rowHeight: InputStyle.rowHeight
   property var projectIssuesModel: ListModel {}
   property string projectLoadingLog: ""
-  property string headerText: ""
+  property string headerText: qsTr( "The following layers failed loading" ) + ":"
```

Comment:
relevant change?

Reply:
Yes, it was previously sent from the Loader